exampleSite/config.toml

```diff
@@ -68,10 +68,8 @@ enableEmoji = true
   textFontFamily = "sans-serif"
   headingFontFamily = "sans-serif"
 
-  # Specify use of utterances or giscus
-  [params.comments]
-    use_utterances = false
-    use_giscus = true
+  # Specify use of utterances, giscus or disqus
+  comments = "utterances"
```

Comment:
This is what you were thinking @gadenbuie ?

Reply:
Yes! 💯

Reply:
thank you both for this- I totally agree ➕